README.md

```diff
@@ -1,15 +1,50 @@
 # qrvr
-SteamVR QR code reader for node.js
+
+SteamVR QR code reader for node.js.
+
+Note: OpenVR is required for qrvr to work. See the error codes below for more details.
 
 ```
 npm install qrvr
 ```
 
 ```
 const qrvr = require('qrvr');
-qrvr.start().then(s => {
+const exitCode = qrvr.start().then(s => {
```

Comment:
I'm not sure of this will have the exit code. Won't this be a `Promise`?

Reply:
I see, so the intent is to capture the exit code. However, this isn't how promises work in ES5.

First, we would probably need to `await` here. Secondly, I don't think the promise itself is resolving to the exit code, or even waiting for the termination of the process (see below).

Reply:
If you log `exitCode` I suspect you will see it is a promise that resolves to a string.

Reply:
I actually didn't test that snippet, my bad sorry. I'll try testing it as soon as I can! My thinking was that because I was `process.exit`ing, and because we're not `.catch`ing the `start` promise, it would just end with that status code but I might be wrong so will check! :)